pkg/fetch/bucket.go

```diff
@@ -46,7 +47,7 @@ func (f *bucketFetcher) File(pathElements ...string) ([]byte, error) {
 		return content, nil
 	}
 
-	if blob.IsNotExist(err) {
+	if gcerrors.Code(err) == gcerrors.NotFound {
```

Comment:
If `gcerrors` is Google Cloud, then this can't be right here

Reply:
It has to, since this is how they refactored it upstream. See this diff: https://github.com/google/go-cloud/pull/1127/files#diff-c334d20eb9e4de3c42c0030530455b56L81 for example

Reply:
OK, so `gcerrors` is `gocloud`. Makes sense then!